experimental/parser/lex_string.go

```diff
@@ -107,19 +107,7 @@ func lexStringContent(q rune, l *lexer) (sc stringContent) {
 		nl := l.SpanFrom(l.cursor - utf8.RuneLen(r))
 		l.Errorf("unescaped newlines are not permitted in string literals").Apply(
 			report.Snippet(nl),
-			report.SuggestEdits(nl, "replace this with an explicit `\\n`",
-				report.Edit{
-					Start:   0,
-					End:     1,
-					Replace: fmt.Sprintf("\\n%c\n", q),
-				},
-				report.Edit{
-					Start:   1,
-					End:     1,
-					Replace: string(q),
-					Justify: report.JustifyRight,
-				},
-			),
+			report.Helpf("consider splitting this into two adjacent string literals"),
```

Comment:
There may be more than one newline, so hard-coding a count of two adjacent literals doesn't seem right.
```suggestion
			report.Helpf("consider splitting this into adjacent string literals"),
```